docs/content/hasher.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "Hasher"
+description: "Handle checksums for other remotes"
+---
+
+# {{< icon "fa fa-check-double" >}}Hasher (EXPERIMENTAL)
+
+The `hasher` overlay caches hashsums for underlying remote.
+
+To use it, first set up the underlying remote following the configuration
+instructions for that remote. You can also use a local pathname instead of
+a remote.
```

Comment:
Could do with a paragraph on why you would use hasher.

Reply:
ok.